src/reference_mapping/scarches/config.vsh.yaml

```diff
@@ -1,31 +1,57 @@
 name: scarches
-namespace: "integrate"
-description: "Performs reference mapping with scArches"
+namespace: "reference_mapping"
```

Comment:
Unfortunatly, this is a breaking change. Could you hold changing the namespace until `3.0.0`? We can still do the other changes.